src/components/Header/index.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { bool, func } from 'prop-types';
+import { makeStyles } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+import Button from '@material-ui/core/Button';
+import { schema } from '../../utils/prop-types';
+
+const useStyles = makeStyles(theme => ({
+  container: {
+    backgroundColor: theme.palette.grey[300],
+    color: theme.palette.text.primary,
+    padding: `${theme.spacing(1.5)}px ${theme.spacing(1)}px`,
+    marginBottom: `${theme.spacing(1)}px`,

```

Comment:
This could just be `marginBottom: theme.spacing(1),`

Reply:
Applied changes!